ui/components/app/wallet-overview/index.scss

```diff
@@ -75,11 +78,6 @@
     max-width: 326px;
   }
 
-  &__icon-button {
-    margin-top: 0 !important;
-    color: var(--color-icon-default);
-  }
```

Comment:
This is the class that was used with the `iconButtonClassName` prop and is no longer needed